### DISCUSSION TITLE
Some modern coloristic examples

### DISCUSSION THREAD

**Discussion by eldinor** [jelster/space-truckers#29] [Ideas]
![image](https://user-images.githubusercontent.com/25833327/106525254-c08a0280-64f4-11eb-8b50-24d8658a9703.png)
![image](https://user-images.githubusercontent.com/25833327/106525278-c97ad400-64f4-11eb-97cd-9fd31447eb98.png)
![image](https://user-images.githubusercontent.com/25833327/106525288-ced81e80-64f4-11eb-962f-fe344a851940.png)
![image](https://user-images.githubusercontent.com/25833327/106525299-d3043c00-64f4-11eb-8fc3-8ecc6c20a80d.png)
![image](https://user-images.githubusercontent.com/25833327/106525310-d7305980-64f4-11eb-97ed-39c792dc1e81.png)
![image](https://user-images.githubusercontent.com/25833327/106525320-dbf50d80-64f4-11eb-9e26-cc3e71840d42.png)
![image](https://user-images.githubusercontent.com/25833327/106525338-e3b4b200-64f4-11eb-85c0-616d71f6bb58.png)
![image](https://user-images.githubusercontent.com/25833327/106525370-ee6f4700-64f4-11eb-825a-884c665308dd.png)
![image](https://user-images.githubusercontent.com/25833327/106525358-e9aa9300-64f4-11eb-9c4e-3c753003a85e.png)
![image](https://user-images.githubusercontent.com/25833327/106525388-f4fdbe80-64f4-11eb-8c98-cdeb61ba7fd3.png)
![image](https://user-images.githubusercontent.com/25833327/106525378-f202ce00-64f4-11eb-953b-30c93a5c35ac.png)
![image](https://user-images.githubusercontent.com/25833327/106525398-f8914580-64f4-11eb-83b9-b17ca0b6966f.png)
![image](https://user-images.githubusercontent.com/25833327/106525409-fb8c3600-64f4-11eb-830b-8ea001b6b0c4.png)

**Comment by jelster** [comment]
Thanks for researching these! I like the Nova Seed color stylings, and I like the "chill" city scape. The Brandt was useful because it was different - didn't fit aesthetically but helped me understand what I wanted from the others! 

I'm a fan of the two-tone shading done in Hades, I guess it's the rich saturated tones?
![image](https://user-images.githubusercontent.com/1508500/106556770-e52aae00-64e5-11eb-9139-3ff16d170629.jpeg)

**Comment by jelster** [comment]
Link to Issue #18

**Comment by jelster** [comment]
[this video](https://youtu.be/K0PXzE1hJXg) and its 2 follow up ones walk through building a toon shader using the NME.

**Reply by jelster** [reply]
Link to final shader https://nme.babylonjs.com/#81NNDY
